src/assets/browser/app/src/controller/global/activate.rs

```diff
@@ -14,7 +14,9 @@ impl EventListener<()> for ReadyPingPongListener {
             EventData::MessageEvent(_,_,c) => {
                 let data = &c.data().unwrap();
                 if data["type"] == "bpane-ready-query" {
-                    domutil::send_post_message("bpane-ready",&json!{{}});
+                    domutil::send_post_message("bpane-ready",&json!{{
```

Comment:
Wouldn't it be `"bpane-out"` rather than `"bpane-ready"`?

Reply:
I tried to change it to `bpane-out` but the problem is that every subscriber will now be notified about this message and they will all have to process the action to find out. For example, a message with type `bpane-ready` is sent only once so that `bpane-ready` subscriber gets notified only once. But it we change this to `bpane-out`, for every single message we receive from the browser, the `bpane-ready` subscriber will also be notified which is unnecessary. 

Considering this, I was wondering if it would be better to keep these types separated which gets used only once.

Reply:
> every subscriber will now be notified

Can't browserMessagingService have a special case for bpane-out with action "genome_browser_ready" and just keep this message to itself without notifying subscribers when it sees it?

Reply:
I mean, `bpane-ready` with action `"genome_browser_ready"` kinda doesn't make sense. We can leave the message type to be bpane-ready, but then there's no point adding a payload with an action to it. Or we could inspect the payload of bpane-out to decide whether to notify subscribers or not.